### PR TITLE
[flang][Driver][test] add missing run lines to fopenmp test

### PR DESCRIPTION
**Pull request by tblah**
I believe these were forgotten when copying the clang in #86816.

This was flagged because the CHECK lines for CHECK-LD-ANY* had no associated RUN line. See
https://github.com/llvm/llvm-project/pull/92387#issuecomment-2119170354

### CODE REVIEW

**Comment by tblah** [comment]
Ping @klensy does this fix the issue you saw in this test? (It won't let me add you as a reviewer unfortunately)

**Comment by klensy** [comment]
It *looks* ok, but CI still pending.

File: flang/test/Driver/fopenmp.f90 [hunk 1]
```diff
@@ -51,9 +51,18 @@
...
 ! to *always* semantically analyze OpenMP without always generating runtime
 ! calls (in the event of an unsupported runtime), we don't have a good way to
-! test the CC1 invocation. Instead, just ensure we do eventually link *some*
+! test the FC1 invocation. Instead, just ensure we do eventually link *some*
 ! OpenMP runtime.
 !
+! RUN: %flang -target x86_64-linux-gnu -fopenmp %s -o %t -### 2>&1 | FileCheck %s --check-prefix=CHECK-LD-ANY
+! RUN: %flang -target x86_64-darwin -fopenmp %s -o %t -### 2>&1 | FileCheck %s --check-prefix=CHECK-LD-ANY
+! RUN: %flang -target x86_64-freebsd -fopenmp %s -o %t -### 2>&1 | FileCheck %s --check-prefix=CHECK-LD-ANY
+! RUN: %flang -target x86_64-netbsd -fopenmp %s -o %t -### 2>&1 | FileCheck %s --check-prefix=CHECK-LD-ANY
```
Comment (Leporacanthicus) [inline flang/test/Driver/fopenmp.f90:60]: We probably don't need to have ALL of these different targets. Very nit-picky, no big deal if we keep them, just trying to keep down the number of permutations being run in tests...

**Comment by Leporacanthicus** [comment]
Looks OK. Minor nit-comment. Feel free to ignore.

**Pull request merged**